terraform/modules/tf_athena/main.tf

```diff
@@ -160,14 +160,15 @@ resource "aws_cloudwatch_log_group" "athena" {
 // CloudWatch metric filters for the athena partition refresh function
 // The split list is made up of: <filter_name>, <filter_pattern>, <value>
 resource "aws_cloudwatch_log_metric_filter" "athena_partition_refresh_cw_metric_filters" {
-  count          = "${length(var.athena_metric_filters)}"
-  name           = "${element(split(",", var.athena_metric_filters[count.index]), 0)}"
-  pattern        = "${element(split(",", var.athena_metric_filters[count.index]), 1)}"
-  log_group_name = "${aws_cloudwatch_log_group.athena.name}"
+  count          = length(var.athena_metric_filters)
+  name           = element(split(",", var.athena_metric_filters[count.index]), 0)
+  pattern        = element(split(",", var.athena_metric_filters[count.index]), 1)
+  log_group_name = aws_cloudwatch_log_group.athena.name
 
   metric_transformation {
-    name      = "${element(split(",", var.athena_metric_filters[count.index]), 0)}"
-    namespace = "${var.namespace}"
-    value     = "${element(split(",", var.athena_metric_filters[count.index]), 2)}"
+    name      = element(split(",", var.athena_metric_filters[count.index]), 0)
+    namespace = var.namespace
+    value     = element(split(",", var.athena_metric_filters[count.index]), 2)
   }
 }
+
```

Comment:
a bit of a nit - but can you update your editor to not add extraneous new lines at the bottom of `.tf` files? I don't want us to have to constantly dismiss changes in the future because our IDEs are trying to _remove_ these lines